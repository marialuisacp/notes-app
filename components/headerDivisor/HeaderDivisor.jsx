import React from 'react';
import StyledHeaderDivisor from './styles/HeaderDivisor.styles';

const HeaderDivisor = () => (
  <StyledHeaderDivisor></StyledHeaderDivisor>
);

export default HeaderDivisor;