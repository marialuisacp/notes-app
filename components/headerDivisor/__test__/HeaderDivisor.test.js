import React from 'react';
import '@testing-library/jest-dom';
import { render } from '@testing-library/react';
import HeaderDivisor from '../HeaderDivisor';

test('should render component HeaderDivisor', () => {
  const { fragment } = render(<HeaderDivisor></HeaderDivisor>);
  expect(fragment).toMatchSnapshot();
});
