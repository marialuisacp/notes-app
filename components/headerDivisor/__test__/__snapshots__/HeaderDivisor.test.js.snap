// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render component HeaderDivisor 1`] = `undefined`;
